index.js

```diff
@@ -167,7 +167,7 @@ exports.decorateConfig = (config) => {
         color: ${TAB_TEXT_COLOR};
         font-weight: 600;
       }
-      ${windowControlsCSS}
+      ${windowControlsCSS || ''}
```

Comment:
Instead of doing this, why don't we initialize the variable to blank. 

https://github.com/weirdpattern/hyper-ayu-mirage/blob/7f540910875bb7dc2fc2dfd7d8ddb42f68aa3e8c/index.js#L75

I find `or` statements inside literal templates hard to read.